llvm/lib/SYCLLowerIR/ESIMD/LowerESIMD.cpp

```diff
@@ -672,12 +672,21 @@ class ESIMDIntrinDescTable {
         {"slm_init", {"slm.init", {a(0)}}},
         {"bf_cvt", {"bf.cvt", {a(0)}}},
         {"tf32_cvt", {"tf32.cvt", {a(0)}}},
+        {"addc", {"addc", {l(0)}}},
```

Comment:
Let's not use GenX intrinsics. Please use SPIRV operation "OpIAddCarry" and "OpISubBorrow":
https://registry.khronos.org/SPIR-V/specs/unified1/SPIRV.html

Reply:
@vmustya - can you please comment/elaborate?
Is there some barrier on the way of using IOpAddCarry and GenX is recommended?

Reply:
As I can see, neither OpIAddCarry nor OpISubBorrow are implemented in the spir-v translator. So we have two options here:
1. Wait until the implementation is ready
2. Implement ESIMD support using VC intrinsic, and re-implement using standard SPIR-V when the the translator is ready

Reply:
Ok, thank you. Let's use 2nd approach for now, then change our internal implementation when SPIRV ext/translator is ready.